### ISSUE TITLE
Session Auto-commit not working or failing silently

### ISSUE THREAD

**Issue by Dschoni** [vertica/vertica-python#232]
I'm trying to set my session to non-auto-commit mode for performance reasons (and the possibility to rollback transaction based).

However executing the command to disable AUTCOMMIT

`cur.execute("SET SESSION AUTOCOMMIT TO off;")`

does not lead to an error, neither does it seem to work as the next statement (e.g. `CREATE TABLE`) is again being auto-commited. Is there another way to disable AUTOCOMMIT session-wide?

**Comment by sitingren** [comment]
Please note that DDL statements are autocommitted. So `CREATE TABLE` wouldn't be affected by this AUTCOMMIT setting. Have you tried other queries like `INSERT`?

**Comment by sitingren** [comment]
@Dschoni I merged a PR #233, so when you turn on logging to DEBUG, you should see the following log when you run `cur.execute("SET SESSION AUTOCOMMIT TO off;")`:
```
<DEBUG> => Query
<DEBUG> <= ParameterStatus: auto_commit = off
<DEBUG> <= CommandComplete
<DEBUG> <= ReadyForQuery
```
`ParameterStatus: auto_commit = off` is the message the database server sends to the client. You can confirm if the auto-commit setting works or not.

**Comment by Dschoni** [comment]
You are 100% right with your above statement. I checked the behaviour with an `INSERT` statement, and that worked as expected.
